rust/src/measurements/private_quantile/mod.rs

```diff
@@ -56,15 +56,19 @@ where
 
     // scale up the noise proportionally to the increase in sensitivity
     // due to the integerization of scores
-    let denominator = score_candidates_constants(input_domain.size.clone(), alpha)?.1;
+    let denominator = score_candidates_constants(
+        input_domain.size.map(u64::exact_int_cast).transpose()?,
+        alpha,
+    )?
+    .1;
 
     let t_score =
         make_quantile_score_candidates(input_domain, input_metric, candidates.clone(), alpha)?;
 
     let m_rnm = make_report_noisy_max_gumbel(
         t_score.output_domain.clone(),
         t_score.output_metric.clone(),
-        scale * denominator as f64 * 2.0,
```

Comment:
I dropped this factor because I ultimately decided that I think it makes things more confusing. Median costs 1 epsilon when scale is 1 at sensitivity 1.